i18next-xhr-backend/i18next-xhr-backend.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for i18next-xhr-backend 1.2.0
+// Project: http://i18next.com/
+// Definitions by: Jan Mühlemann <https://github.com/jamuhl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'i18next-xhr-backend' {
+
+    interface Interpolator {
+        interpolate: () => string
+    }
+    interface Services {
+        interpolator: Interpolator
+    }
+
+    export default class Backend {
+        type: 'backend';
+        services: Services;
+        options: {
+            loadPath: string | Function,
+            addPath:  string,
+            allowMultiLoading: boolean,
+            parse: () => {},
+            crossDomain: boolean,
+            withCredentials: boolean,
+            ajax: () => void
+        };
+        constructor(services: Services, options?: {});
```

Comment:
`services` can be optional.
See example from [README.md](https://github.com/i18next/i18next-xhr-backend#backend-options).